civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +72,192 @@
     }
   }
 }
+
+.view-mydetails-personal {
```

Comment:
The following styles from line 76 to 263 should be moved to `civihr_default_theme/scss/drupal/_view.scss`.

Reply:
@deb1990 why? The context in which this view appears is in the context of my details page, are you sure? I think different but if you are sure that it is the best thing to do I will move it.

Reply:
okay in this case just keep it. Normally they should go to views, but as this is a POC, its fine.